app/controllers/ideas_controller.rb

```diff
@@ -1,7 +1,10 @@
 class IdeasController < ApplicationController
-  respond_to :html, :json
+  respond_to  :json, :html
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.